packages/renderer/src/lib/container/ContainerList.svelte

```diff
@@ -341,8 +341,8 @@ let ageColumn = new TableColumn<ContainerInfoUI | ContainerGroupInfoUI, Date | u
     return undefined;
   },
   comparator: (a, b) => {
-    const aTime = containerUtils.isContainerInfoUI(a) ? (moment().diff(containerUtils.getUpDate(a)) ?? 0) : 0;
-    const bTime = containerUtils.isContainerInfoUI(b) ? (moment().diff(containerUtils.getUpDate(b)) ?? 0) : 0;
+    const aTime = containerUtils.isContainerInfoUI(a) && a.state === 'RUNNING' ? (moment().diff(a.startedAt) ?? 0) : 0;
```

Comment:
Why only RUNNING. A stopped container may have an age as well ?

Reply:
@jeffmaury only the age of running containers is displayed in the containers list table. Should I sort them based on only seen age or also unseen age?

Reply:
That needs to be discussed but not on this PR

Reply:
@jeffmaury remove the running part for this PR?

Reply:
No keep it as I'm discussing why age is not displayed for stopped containers but that's another PR